app/containers/EditRequestPage.js

```diff
@@ -14,6 +14,7 @@ class EditRequestPage extends Component {
 
   onExecuteRequest(request) {
     this.props.executeRequest(request.method, request.url, request.headers.toJS(), request.body);
+    this.props.logRequest(request);
```

Comment:
initially I had it there (more precisely [here](https://github.com/ruisalgado/pragma/commit/d53e9d04d58d5745cb38d14d5ab97b4800296610#diff-a75d1711078c51fae2354ba4255a6f9bL137)) but I moved it out. It felt a bit dirty - I was actually expecting to see a EXECUTE_REQUEST action that I could also subscribe to with the history story (no LOG_REQUEST action would be needed). At the time I was also not familiar with redux-thunk so it felt extra dirty